vector/src/main/java/im/vector/app/features/VectorFeatures.kt

```diff
@@ -24,19 +24,17 @@ interface VectorFeatures {
 
     fun isAlreadyHaveAccountSplashEnabled(): Boolean
 
+    fun isSplashCarouselEnabled(): Boolean
+
     enum class OnboardingVariant {
         LEGACY,
         LOGIN_2,
         FTUE_AUTH
     }
-
-    enum class NotificationSettingsVersion {
-        V1,
-        V2
-    }
 }
 
 class DefaultVectorFeatures : VectorFeatures {
     override fun onboardingVariant(): VectorFeatures.OnboardingVariant = BuildConfig.ONBOARDING_VARIANT
     override fun isAlreadyHaveAccountSplashEnabled() = true
+    override fun isSplashCarouselEnabled() = false
```

Comment:
whilst the carousel is still WIP it's not enabled by default for the `FTUE_AUTH` variant